src/ICSharpCode.SharpZipLib/Tar/TarArchive.cs

```diff
@@ -139,15 +167,15 @@ public static TarArchive CreateInputTarArchive(Stream inputStream, int blockFact
 				throw new ArgumentException("TarInputStream not valid");
 			}
 
-			return new TarArchive(new TarInputStream(inputStream, blockFactor));
+			return new TarArchive(new TarInputStream(inputStream, blockFactor, nameEncoding));
 		}
-
 		/// <summary>
 		/// Create a TarArchive for writing to, using the default blocking factor
 		/// </summary>
 		/// <param name="outputStream">The <see cref="Stream"/> to write to</param>
+		/// <param name="nameEncoding"></param>
```

Comment:
not trying to do a real review (not up to speed with the changes/requirements), but fwiw - empty doc comment for the new param?

Reply:
There were comfirmation omission. I inserted comment.

Reply:
Yeah, I think that the variable name here does enough to explain the purpose. I see little to no gain in writing descriptions like "The Encoding used for the name".

Reply:
Sorry for late response.
Should I leave this parameter comment empty?

Reply:
It's fine